pyLibrary/queries/expressions.py

```diff
@@ -1768,11 +1768,14 @@ def to_python(self, not_null=False, boolean=False, many=False):
         return "re.match(" + quote(json2value(self.pattern.json) + "$") + ", " + self.var.to_python() + ")"
 
     def to_sql(self, schema, not_null=False, boolean=False):
-        pattern = schema.db.quote_value(convert.json2value(self.pattern.json))
+        pattern = quote(convert.json2value(self.pattern.json))
         value = self.var.to_sql(schema)[0].sql.s
-        return wrap([
-            {"name": ".", "sql": {"s": value + " REGEXP " + pattern}}
-        ])
+        if not boolean:
+            return wrap([
+                {"name": ".", "sql": {"s": value + " REGEXP " + pattern}}
```

Comment:
This should be `{"name": ".", "sql": {"b": value + " REGEXP " + pattern}}` because it returns boolean